### DISCUSSION TITLE
Sorting using createdBy attribute

### DISCUSSION THREAD

**Discussion by MerNat** [knoxpo/dart_algolia#109] [Q&A] [answered]
Is there a way to sort out a query by the attribute of `createdBy` as I'm trying to get the newly added items first in my result?

Any suggestion or idea on how to add the setting/subquery to this statement:
```
      AlgoliaQuery _query = _algolia.instance
          .index("text-index")
          .setPage(startingKey!)
          .setHitsPerPage(pageSize);
```

**Comment by muntasir-ifaz574** [comment, marked as answer]
AlgoliaQuery _query = _algolia.instance
    .index("text-index")
    .setPage(startingKey!)
    .setHitsPerPage(pageSize)
    .setSettings({
      'customRanking': ['desc(createdBy)']
    });
